test/functionalTest/cases/501_cors/get_any_origin.test

```diff
@@ -0,0 +1,134 @@
+# Copyright 2014 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
@jmcanterafonseca: have specially a look to the .test files included in this PR in order to check if the CB, after being started with SHELL-INIT and feed with what is shown in SHELL section should return the output shown in REGEXPECTED. 

I think the syntax of the test specification is quite straighforward, but if you have doubts, don't hesitate to contact me offline.